### PR TITLE
Improve installation instructions

### PR DESCRIPTION
**Pull request by tijmenb**
[Rendered](https://github.com/alphagov/govuk-docker/blob/improve-installation-instructions/README.md#setup)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,28 +38,26 @@ However, this high-level statement hides a great number of specific needs, which
...
-   - A directory `~/govuk` in your home directory
+1. [dnsmasq](http://www.thekelleys.org.uk/dnsmasq/doc.html) to make *app-name.dev.gov.uk* work (`brew install dnsmasq`)
+2. [docker](https://hub.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/)
+3. [git](https://git-scm.com) if you're setting everything up from scratch
+4. A directory `~/govuk` in your home directory
+
+### 2. Configure shell
 
-Start with the following in your bash config (aliases optional).
+Start with the following in your `.bash_profile`:
```
Comment (thomasleese) [inline README.md]:
```suggestion
Start with the following in your `~/.bash_profile`:
```

File: README.md [hunk 2]
```diff
@@ -38,28 +38,26 @@ However, this high-level statement hides a great number of specific needs, which
...
 ```
 export PATH=$PATH:~/govuk/govuk-docker/bin
 ```
 
-Now in the `govuk` directory, run the following commands.
-
-```
-git clone git@github.com:alphagov/govuk-docker.git
-cd govuk-docker
+Reload your shell with `source .bash_profile`.
```
Comment (thomasleese) [inline README.md]:
```suggestion
Reload your shell with `source ~/.bash_profile`.
```

Comment (thomasleese) [reply README.md]: In case someone doesn't happen to be in `~` when they run `source`.

**Comment by tijmenb** [comment]
I'll let the setup team

**Pull request closed without merging**